src/jemalloc/src/util.c

```diff
@@ -344,7 +344,7 @@ malloc_vsnprintf(char *str, size_t size, const char *format, va_list ap)
 	}								\
 } while (0)
 #define	GET_ARG_NUMERIC(val, len) do {					\
-	switch (len) {							\
+	switch ((size_t)len) {							\
```

Comment:
you are in a macro, so you have to surround arguments with () before doing any operations

why size_t? wouldn't unsigned or int be enough?